packages/extension-chakra-storefront/src/components/register/index.jsx

```diff
@@ -41,7 +41,9 @@ const RegisterForm = ({submitForm, clickSignIn = noop, form}) => {
                 <Stack paddingTop={8} gap={8} paddingLeft={4} paddingRight={4}>
                     {form.formState.errors?.global && (
                         <Alert.Root status="error">
```

Comment:
Quesiton: Are we missing migrating the Alert component for the Login form?

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/701fb4de9a7118692f103e5744eafaf576ff562d/packages/extension-chakra-storefront/src/components/login/index.jsx#L50-L57

Reply:
Another nit: The Login form is missing migrating `spacing` to `gap`:

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/701fb4de9a7118692f103e5744eafaf576ff562d/packages/extension-chakra-storefront/src/components/login/index.jsx#L58

Reply:
Ah thank you, Adam.. will fix them.